libwayshot/src/lib.rs

```diff
@@ -174,18 +178,19 @@ impl WayshotConnection {
         cursor_overlay: i32,
         output: &WlOutput,
         fd: T,
-        capture_region: Option<CaptureRegion>,
-    ) -> Result<FrameFormat> {
+    ) -> Result<(FrameFormat, FrameGuard)> {
```

Comment:
I have afraid this will be a breaking change, and before, this api is used for screen cast, and in screen cast, frameguard is not needed

Reply:
True, I haven't addressed backwards compatibility as this is only a draft PR. It'll be addressed once the PR is ready.